include/zephyr/drivers/zephyr_thermistor.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_THERMISTOR_H
+#define ZEPHYR_THERMISTOR_H
+
+#include <zephyr/types.h>
+
+struct ntc_compensation {
```

Comment:
@asemjonovs - can I suggest just making both of these fields 32-bit? That would make initialization from DT macro 
straightforward.
https://docs.zephyrproject.org/3.1.0/build/dts/bindings.html#property-types

Reply:
Sorry - just noticed that was already suggested.